src/chart/treemap/Breadcrumb.ts

```diff
@@ -185,7 +189,11 @@ class Breadcrumb {
                 onclick: curry(onSelect, itemNode)
             });
             (el as ECElement).disableLabelAnimation = true;
-
+            el.getTextContent().ensureState('emphasis').style = {text, ...createTextStyle(emphasisTextStyleModel)};
+            el.ensureState('emphasis').style = emphasisModel.getModel('itemStyle').getItemStyle();
+            toggleHoverEmphasis(
+                el, emphasisModel.get('focus'), emphasisModel.get('blurScope'), emphasisModel.get('disabled')
```

Comment:
Does the focus/blur state work? I think it may need some extra work.

Reply:
No it doesn't work, I just add it for the future

Reply:
Just notice that current model getters are in the for loop, I'm not sure if we should cache them to local variables(Acceptable for me). I think it is up to you. @susiwen8 @pissang